src/assets/fonts.tsx

```diff
@@ -0,0 +1,11 @@
+import { injectGlobal } from 'styled-components'
+
+injectGlobal`
+    @font-face {
+        font-family: 'artsy-icons';
```

Comment:
Since we're paving the future here, I'll throw my 👎  to icon fonts and 👍  to rendering SVGs directly for a couple reasons:

1. Icon fonts are a PITA to update b/c you often have to go through some [complicated build process](https://github.com/artsy/artsy-iconfont#managing-the-font) to turn the source files into fonts
2. You don't have to have the overhead of making mapping font character to css class name—just import SVG file basically
3. They involve un-intuitive properties like using `font-size` to change the size of them
4. Although a more rare use case, you don't have fine-grain control of the SVG like being able to change things like `stroke` or animate color `fill`

The nicest way I've found to do this is to use `fs.readFileSync` and shim `fs` in [the browser](https://github.com/substack/brfs), but there's [other](https://github.com/necinc/msvgc) [tools](http://fuse-box.org/#loader-api) that do similar things—the important part is that something takes SVG file in and puts JS out. You can even [do this manually](https://github.com/artsy/team-navigator/blob/master/app/views/sidebar/artsy-logo.svg.js) to avoid build/tools plugins.